### DISCUSSION TITLE
Password change and number of script-run requests.

### DISCUSSION THREAD

**Discussion by zxcvryo** [chriskyfung/AutoFetcher-IG-Stories-to-GDrive#66] [Q&A]
I just wanted to know if changing the Instagram password (the account from which I fetched the values for the 'settings' tab in the google spreadsheet file) of the account hinders the script in any way. I wanted to know because Instagram made me change my password because of a 'phishing' alert after I ran the script for the first time.

Also, how many requests are possible per day to run the script? I set it to every 10 minutes. Hope that's not much.

Also Idk why the health status svg shows failed, I did everything right and rechecked it several times.

**Comment by chriskyfung** [comment]
Please refer to Issue #45 .
